ietf-ns-topo.yang

```diff
@@ -312,10 +349,9 @@
              "/ietf-nss:steering-constraints" +
              "/ietf-nss:path-constraints" {
        description
-         "Add toplogy id and explicit path to the connectivity
-		  construct";
+         "Underlay path for connection group.";
```

Comment:
connection-group ? not connectivity-construct ..?

Reply:
This augment is for connection group, which applies to all connectivity-constructs with the group. 

There is a separate augment for connectivity constructs, which applies to a single connectivity construct.

The two has different scope.